larray/core/array.py

```diff
@@ -10285,12 +6621,7 @@ def read_csv(filepath_or_buffer, nb_index=None, index_col=None, sep=',', headers
         kwargs['header'] = 1
         kwargs['comment'] = '#'
 
-    if nb_index is not None and index_col is not None:
-        raise ValueError("cannot specify both nb_index and index_col")
-    elif nb_index is not None:
-        index_col = list(range(nb_index))
-    elif isinstance(index_col, int):
-        index_col = [index_col]
+    index_col = _get_index_col(nb_index, index_col)
```

Comment:
please move this change (factorization of _get_index_col) to another PR. It is a good change, but such a large PR should only contain what is strictly necessary, otherwise extra stuff like this get completely lost/can go unnoticed.